tests/virt_autotest/transactional/prepare_transactional_server.pm

```diff
@@ -69,7 +75,7 @@ sub prepare_packages {
         # should not prevent user from installing legitimate tools and utilities, it
         # is expected that use of additional packages should be limited to the minimum
         # and their impact should be analyzed beforehand.
-        my @repos_to_install = split(/,/, get_var("INSTALL_OTHER_REPOS"));
+        my @repos_to_install = split(/,/, $additional_repos);
```

Comment:
Why this change ?

Reply:
I extract the common part as a function 'install_additional_pkgs()' for leverage in kubevirt tests with sle-micro. Actually, the variable 'INSTALL_OTHER_REPOS' is invoked 2 times, just want to state it at beginning to make it more explicit.
`if (get_var("INSTALL_OTHER_REPOS")) {`

Reply:
It does not solve "calling twice" problem. In fact, any variable in a function will be called once when the function is called. So it looks to me this change is unnecessary.

Reply:
I'm ok if you'd like to keep the original code, no any impact for me.